timesheetLogic.js

```diff
@@ -99,3 +70,53 @@ database.ref('/timesheet').on("child_added", function(childSnapshot, prevChildKe
 
 // We know that this is 15 months.
 // Now we will create code in moment.js to confirm that any attempt we use mets this test case
+
+function appendEmp (snap) {
+  var emp = snap.val()
+  var row = $('<tr>')
+  row.append($('<td>').text(emp.name))
+  row.append($('<td>').text(emp.role))
+  row.append($('<td>').text(emp.rate))
+
+  var next = $('<td>')
+  var away = $('<td>')
+  row.append(next)
+  row.append(away)
+  row.on('click', removeRow)
+
+  $("#train-table > tbody").append(row)
```

Comment:
This is the main change #employee-table should be #train-table thats why the data was never showing up in the table